### ISSUE TITLE
Multisite always return the same

### ISSUE THREAD

**Issue by cmartinespinosa** [putyourlightson/craft-campaign#278]
### Describe the bug
I have problem when I use craft-campaign with multisite because always return the same MailingList. I was able to solve it modifying the getMailingListBySlug method and commented the line `->siteId($siteId)`

**putyourlightson/craft-campaign/src/services/MailingListsService.php (line: 89)**

```
public function getMailingListBySlug(string $mailingListSlug)
    {
        // Get site ID from element site settings
        $siteId = Element_SiteSettings::find()
            ->select('siteId')
            ->where(['slug' => $mailingListSlug])
            ->scalar();

        if ($siteId === null) {
            return null;
        }

        return MailingListElement::find()
            ->slug($mailingListSlug)
            //->siteId($siteId)
            ->one();
    }
```

### Versions
- Plugin version: 1.20.2
- Craft version: 3.6.4

**Comment by bencroker** [comment]
Thanks, this should be fixed in https://github.com/putyourlightson/craft-campaign/commit/e5e6f74e2511031d4cdfe6729940f99fd658abf7. Feel free to test and let me know if that solves the issue for you.

**Comment by bencroker** [comment]
Released in version 1.22.0.
